movo_network/movo_network_config.bash

```diff
@@ -18,14 +18,15 @@ if [ "$HOSTNAME" = movo1 ]; then
     export ROBOT_NETWORK=enp0s25
     export ROS_IP=$(ip -4 address show $ROBOT_NETWORK | grep 'inet' | sed 's/.*inet \([0-9\.]\+\).*/\1/' | head -n 1)
     export ROS_MASTER_URI=http://movo2:11311/
+    export KINOVA_MOVO1_IP=10.66.171.1
```

Comment:
looks like u forgot to add it for MOVO2 but id recommend deleting this anywau see prev comment